src/d3d8/d3d8_device.cpp

```diff
@@ -1168,7 +1168,7 @@ namespace dxvk {
           UINT             PrimitiveCount) {
     return GetD3D9()->DrawIndexedPrimitive(
       d3d9::D3DPRIMITIVETYPE(PrimitiveType),
-      m_baseVertexIndex, // set by SetIndices
+      static_cast<INT>(std::min(m_baseVertexIndex, static_cast<UINT>(INT_MAX))), // set by SetIndices
```

Comment:
I'd prefer `std::numeric_limits<uint32_t>::max()` but the climits one is fine too.

Reply:
I mean, that will ultimately return the climits macro anyway, so meh: https://en.cppreference.com/w/cpp/types/numeric_limits/max

Reply:
Yeah but it's more consistent with the rest of DXVK.
But like I said, I think the other version is fine too.